tests/test_shell_interactive.py

```
@@ -1,70 +1,126 @@
+    process.stdin.write(shell_input)
+    process.stdin.flush()
+    lines_count = 0
+    output = ""
+    while True:
+        output_current = process.stdout.readline()
+        output += output_current
+        lines_count += 1
+        if stop_read_pattern is not None and stop_read_pattern in output_current:
+            break
+        elif lines_to_read is not None and lines_to_read == lines_count:
+            break
```

Ruff

Combine `if` branches using logical `or` operator

Combine `if` branches

(SIM114)